src/pages/LandingPages/CorporateCampaign/CCLandingPage.vue

```diff
@@ -131,7 +131,6 @@
 			</kv-lightbox>
 
 			<kv-lightbox
-				v-if="checkoutVisible"
```

Comment:
I think there will be unintended consequences of allowing this into the DOM all the time. It will likely start to initialize the checkout before we have all the information for that process. It will also load up more async code on page load instead of on demand.

Reply:
done